guard/src/commands/github_pull_test.rs

```diff
@@ -0,0 +1,14 @@
+use super::super::*;
+use super::*;
+
+#[test]
```

Comment:
We would need more unit tests, think of these tests in below categories:

1. check whether methods work as they should with different combinations of inputs passed to them - the tests should show different behaviors being handled, and name of the test should reflect what behavior is expected
2. make tests complimentary, they should be clear and concise and should tests a very small part of your code such as a method
3. check for failures, exceptions in case of invalid input, write negative unit tests for these
4. check for boundary conditions, think of scenarios that may exist and are specific to your code - such as
    1. what would happen if credentials passed are incorrect?
    2. what happens in case the repository doesn't exist?
    3. what would happen in case user's API token does not have access to the repo?